ipfx/dataset/ephys_data_set.py

```diff
@@ -92,47 +127,75 @@ def filtered_sweep_table(self,
 
         if stimuli:
             mask = st[self.STIMULUS_CODE].apply(
-                self.ontology.stimulus_has_any_tags, args=(stimuli,), tag_type="code")
```

Comment:
Are you sure you want to touch this? Are you confident that this will not break something?

Reply:
nope, this is a mistake (I was printing here and deleted too much).

Reply:
oh wait I thought this was in stimulus. This is fine, I just inserted line breaks.